### ISSUE TITLE
Review email template for Contact Page

### ISSUE THREAD

**Issue by ryanfchase** [hackforla/311-data#1754]
### Overview
We need to review the copy and design of the email that is sent out when a user makes a submission on the Contact Page in order to be compliant and on-brand with 311-Data and Hack for LA.

### Action Items

- [x] PM team to sign off on the copy (see below)
  - [x] review subject
  - [x] review message body
- [x] Design lead to consider design implementations for this email
  - [x] see if we have email design templates in Figma/GDrive
  - [x] consider creating a basic email design template for 311-Data

#### 311 Data Contact Page Reply E-mail
Subject
```
311-Data thanks you for your input!
```

Message Body (updated per feedback)
```
{{to_name}},

We appreciate you taking the time to leave a suggestion for our 311Data service request map. Your feedback is important to us, and we are always looking for ways to improve our data visualization tool. Please follow our [LinkedIn](https://www.linkedin.com/company/hack-for-la/) for updates on this civic tech project and others within the Hack for LA organization.

Sincerely,

311Data Volunteers at Hack for LA
```

<details><summary>Previous version of message body</summary>
<p>

```
{{to_name}},

Thank you for leaving a suggestion with us for our 311-Data service request map. We are always looking for ways to improve civic engagement through data visualization. Please follow our [LinkedIn](https://www.linkedin.com/company/hack-for-la/) for updates on this project and others within the Hack For LA organization.

Sincerely,

Volunteers of 311-Data at Hack for LA
```

</p>
</details>

**Comment by bberhane** [comment]
Suggested email copy below:

Subject

`Thank you for your input!`

Message Body

`{{to_name}},`

`We appreciate you taking the time to leave a suggestion for our 311Data service request map. Your feedback is important to us, and we are always looking for ways to improve our data visualization tool. Please follow our [LinkedIn](https://www.linkedin.com/company/hack-for-la/) for updates on this civic tech project and others within the Hack for LA organization.`

`Sincerely,`

`311Data Volunteers at Hack for LA`

**Comment by annaseulgi** [comment]
assigning myself for the action items

**Comment by ryanfchase** [comment]
I spoke with @annaseulgi and we confirmed we can move forward with making a ticket for a designer to work on the email template. Unassigning, will close this later tonight unless something comes up

**Comment by ryanfchase** [comment]
> Suggested email copy below:
> 
> Subject
> 
> `Thank you for your input!`
> 
> Message Body
> 
> `{{to_name}},`
> 
> `We appreciate you taking the time to leave a suggestion for our 311Data service request map. Your feedback is important to us, and we are always looking for ways to improve our data visualization tool. Please follow our [LinkedIn](https://www.linkedin.com/company/hack-for-la/) for updates on this civic tech project and others within the Hack for LA organization.`
> 
> `Sincerely,`
> 
> `311Data Volunteers at Hack for LA`

@bberhane I've made the requested change, thank you!

**Comment by ryanfchase** [comment]
Now that we've discussed the "311 Data" vs "311Data" debate, I shall update the text in the template and then close the issue

**Comment by ryanfchase** [comment]
Alright that's been updated.

<details><summary>Screenshot of version</summary>
<p>

![image](https://github.com/hackforla/311-data/assets/6414668/f67074cf-3b75-4377-97e5-e01934b878d0)


</p>
</details>

**Comment by bberhane** [comment]
Let's remove the hyphen in the subject line in 311-Data

**Comment by ryanfchase** [comment]
Oh shoot -- the subject line didn't update properly. I'll fix that.

**Comment by ryanfchase** [comment]
> Let's remove the hyphen in the subject line in 311-Data

No need for changing hyphen as I've implemented your suggestion from earlier.

<details><summary>Screenshot of subject line</summary>
<p>

![image](https://github.com/hackforla/311-data/assets/6414668/1a1cc390-96da-4991-8bdb-89fc5e21332c)


</p>
</details>

**Comment by annaseulgi** [comment]
@ryanfchase @bberhane is any of this needing to be incorporated into the new email template ticket (ie the body/message)? 
if this needs to be closed, should i be doing anything with this since it has the "ready for design lead" tag. thanks for your patience - i'm still getting accustomed to the milestone/tagging/handing off system!

**Comment by ryanfchase** [comment]
@annaseulgi 

>  is any of this needing to be incorporated into the new email template ticket (ie the body/message)?

I've gone ahead and included the subject/body in the [new email template ticket](https://github.com/hackforla/311-data/issues/1763) for reference -- the main ask of that ticket is really just provide a design for all emails, regardless of the actual body. I'll reiterate this on that ticket just so everyone's clear.

> if this needs to be closed, should i be doing anything with this since it has the "ready for design lead" tag.

Thanks for checking, I sorta jumped the gun and checked off these boxes on your behalf, hope that's alright! Next time, we'll have you remove the `ready for design lead` label as soon as you've completed any action items that are design lead specific. I'll do it now so there's no confusion.

For reference... I checked these off after we chatted on Slack:
> - [x] Design lead to consider design implementations for this email
>   - [x] see if we have email design templates in Figma/GDrive
>   - [x] consider creating a basic email design template for 311-Data

. . .

> still getting accustomed to the milestone/tagging/handing off system!

Yeah thank you! If our formatting requests become too verbose or confusing, feel free to just ask that we go over it in an upcoming meeting.
